pkg/ebpf/controlplane/processes.go

```diff
@@ -2,123 +2,82 @@ package controlplane
+	errs = append(errs, err)
 
 	// Parent (the parent of the thread group leader, no matter if child is a process or LWP)
-
 	parentTid, err := parse.ArgVal[int32](args, "parent_tid")
-	if err != nil {
-		return errfmt.Errorf("error parsing parent_tid: %v", err)
-	}
+	errs = append(errs, err)
 	parentNsTid, err := parse.ArgVal[int32](args, "parent_ns_tid")
-	if err != nil {
-		return errfmt.Errorf("error parsing parent_ns_tid: %v", err)
-	}
+	errs = append(errs, err)
 	parentPid, err := parse.ArgVal[int32](args, "parent_pid")
-	if err != nil {
-		return errfmt.Errorf("error parsing parent_pid: %v", err)
-	}
+	errs = append(errs, err)
 	parentNsPid, err := parse.ArgVal[int32](args, "parent_ns_pid")
-	if err != nil {
-		return errfmt.Errorf("error parsing parent_ns_pid: %v", err)
-	}
+	errs = append(errs, err)
 	parentStartTime, err := parse.ArgVal[uint64](args, "parent_start_time")
-	if err != nil {
-		return errfmt.Errorf("error parsing parent_start_time: %v", err)
-	}
+	errs = append(errs, err)
 
 	// Thread Group Leader (might be the same as the "child", if "child" is a process)
-
 	leaderTid, err := parse.ArgVal[int32](args, "leader_tid")
-	if err != nil {
-		return errfmt.Errorf("error parsing leader_tid: %v", err)
-	}
+	errs = append(errs, err)
 	leaderNsTid, err := parse.ArgVal[int32](args, "leader_ns_tid")
-	if err != nil {
-		return errfmt.Errorf("error parsing leader_ns_tid: %v", err)
-	}
+	errs = append(errs, err)
 	leaderPid, err := parse.ArgVal[int32](args, "leader_pid")
-	if err != nil {
-		return errfmt.Errorf("error parsing leader_pid: %v", err)
-	}
+	errs = append(errs, err)
 	leaderNsPid, err := parse.ArgVal[int32](args, "leader_ns_pid")
-	if err != nil {
-		return errfmt.Errorf("error parsing leader_ns_pid: %v", err)
-	}
+	errs = append(errs, err)
 	leaderStartTime, err := parse.ArgVal[uint64](args, "leader_start_time")
-	if err != nil {
-		return errfmt.Errorf("error parsing leader_start_time: %v", err)
-	}
+	errs = append(errs, err)
 
 	// Child (might be a process or a thread)
-
 	childTid, err := parse.ArgVal[int32](args, "child_tid")
-	if err != nil {
-		return errfmt.Errorf("error parsing child_tid: %v", err)
-	}
+	errs = append(errs, err)
 	childNsTid, err := parse.ArgVal[int32](args, "child_ns_tid")
-	if err != nil {
-		return errfmt.Errorf("error parsing child_ns_tid: %v", err)
-	}
+	errs = append(errs, err)
 	childPid, err := parse.ArgVal[int32](args, "child_pid")
-	if err != nil {
-		return errfmt.Errorf("error parsing child_pid: %v", err)
-	}
+	errs = append(errs, err)
 	childNsPid, err := parse.ArgVal[int32](args, "child_ns_pid")
-	if err != nil {
-		return errfmt.Errorf("error parsing child_ns_pid: %v", err)
-	}
+	errs = append(errs, err)
 	childStartTime, err := parse.ArgVal[uint64](args, "child_start_time")
-	if err != nil {
-		return errfmt.Errorf("error parsing child_start_time: %v", err)
+	errs = append(errs, err)
+
+	// Handle errors
+	for _, err := range errs {
+		if err != nil {
+			return err
+		}
 	}
```

Comment:
Using this approach we loose the specific error message (e.g. "error parsting child_pid...") which will make debugging harder

Reply:
Argval tells the name of the argument and its type whenever there is an error.